blog/2020-03-01-CombiningBPMNWithOptaplanner.adoc

```diff
@@ -0,0 +1,611 @@
+= Combining BPMN with Optaplanner
```

Comment:
That's a boring title.
Sorry to be so blunt, but what I mean is: the title is very important - it's the difference between 10 views and 10 000 views.
A couple of things to note on this title:
- the P of OptaPlanner needs to be capitalized (in filename too). One can argue if that was a good decision 7 years ago, but it it what it is.
- BPMN on its own kinda has a bad reputation: it didn't deliver on its promises. Let use prefix it: "workflow BPMN" or "process BPMN" or maybe just "workflow process" (no BPMN). We should have the BPMN keyword in the article of course.
- "Combine" = boring verb.
Use a title that talks about the potential, the benefit, what you can do with it. Something that inspires, something that poses a question but doesn't answer it. Something catchy. See newspaper article titles, they 're never boring.

Reply:
Some idea's:
- Inject AI into your workflow processes with Kogito jBPM and OptaPlanner
- Empower your workflow processes with intelligence scheduling
- Kogito jBPM + OptaPlanner: Run a flight booking process with optimal seat assignment